conans/test/functional/generators/virtualenv_test.py

```diff
@@ -169,3 +174,22 @@ def environment_test(self):
                "BCKW_SLASH": "old_BCKW_SLASH"}
         with tools.environment_append(env):
             self.basic_test(posix_empty_vars=False)
+
+    @unittest.skipIf(platform.system() == "Windows", "Needs bash terminal")
```

Comment:
bash is neither terminal nor terminal emulator, it's shell

Reply:
also, probably more correct check would be `tools.which("bash") is not None`, as it's possible to have bash on Windows (MSYS, Cygwin), as well as there are Unix systems with other shells besides bash.